transcom/mymove
# MAIN B-20109, B-20111, and B-20112 refactor DocumentUploader, OrdersPanel and OrdersViewer

> [!NOTE]  
> This `main` PR contains three different `integrationTesting` PRs to avoid excessive builds in CircleCI since it is 100% deletion of tech debt and no added functionality

[B-20109 INTEGRATION PR](https://github.com/transcom/mymove/pull/13045)
## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20109)

## Summary

We are not wanting to use `redux-form` and instead use Formik due to library support. This backlog item addresses the `DocumentUploader` component and its use of `redux-form` as well as accessory files.

Luckily, this component isn't being used ANYWHERE in the code base and was tech debt. Removing the file and placing it very gently six feet under should complete the task.

### How to test

1. In VSCode, search for `redux-form`
2. You should no longer see it being used in the `DocumentUploader` component
3. Confirm that `DocumentUploader` no longer exists in the code base and that it isn't being used anywhere else
![Screenshot 2024-06-13 at 2 45 30 PM](https://github.com/transcom/mymove/assets/136510600/60829ff3-2665-43cc-8835-b087280eaf0c)


<br><br>
[B-20111 INTEGRATION PR](https://github.com/transcom/mymove/pull/13046)
## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20111)

## Summary

We are not wanting to use `redux-form` and instead use Formik due to library support. This backlog item addresses the `OrdersPanel` component and its use of `redux-form` as well as accessory files.

Luckily, this component isn't being used ANYWHERE in the code base and was tech debt. Removing the file and related functions used in the component that are no longer necessary should fix this pickle.

### How to test

1. In VSCode, search for `redux-form`
2. You should no longer see it being used in the `OrdersPanel` component
3. Confirm that `OrdersPanel` no longer exists in the code base and that it isn't being used anywhere else
![Screenshot 2024-06-13 at 2 45 53 PM](https://github.com/transcom/mymove/assets/136510600/062f54cd-dc4e-4da0-8b9c-74fd8b0a4824)


<br><br>
[B-20112 INTEGRATION PR](https://github.com/transcom/mymove/pull/13047)
## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20112)

## Summary

We are not wanting to use `redux-form` and instead use Formik due to library support. This backlog item addresses the `OrdersViewerPanel` component and its use of `redux-form` as well as accessory files.

Luckily, this component isn't being used ANYWHERE in the code base and was tech debt. Removing the file and related component and functions used in the components that are no longer necessary should achieve the goal of ridding the earth of `redux-form`.

### How to test

1. In VSCode, search for `redux-form`
2. You should no longer see it being used in the `OrdersViewerPanel` component
3. Confirm that `OrdersViewerPanel` no longer exists in the code base and that it isn't being used anywhere else
![Screenshot 2024-06-13 at 2 46 18 PM](https://github.com/transcom/mymove/assets/136510600/639393ea-5817-49ee-b8dd-0f632f00621f)
